python-checks/src/test/resources/checks/cdk/unrestrictedAdministration_Connections.py

```diff
@@ -0,0 +1,145 @@
+        aws_cdk.aws_codebuild.PipelineProject().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_codebuild.Project().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_rds.DatabaseInstance().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_rds.DatabaseInstanceReadReplica().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_rds.DatabaseCluster().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_rds.ServerlessClusterFromSnapshot().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_rds.DatabaseProxy().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_rds.DatabaseInstanceFromSnapshot().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_rds.ServerlessCluster().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_rds.DatabaseClusterFromSnapshot().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_lambda_nodejs.NodejsFunction().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_fsx.LustreFileSystem().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_ec2.BastionHostLinux().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_ec2.ClientVpnEndpoint().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_ec2.Instance().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_ec2.LaunchTemplate().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_ec2.SecurityGroup().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_kinesisfirehose_alpha.DeliveryStream().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_stepfunctions_tasks.SageMakerCreateTrainingJob().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_stepfunctions_tasks.SageMakerCreateModel().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_stepfunctions_tasks.EcsRunTask().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_redshift_alpha.Cluster().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
+        aws_cdk.aws_opensearchservice.Domain().connections.allow_from(ec2.Peer.any_ipv4(), ec2.Port.all_tcp()) # Noncompliant
```

Comment:
do we need to test them all?

Reply:
To be honest it helped me to make sure there was no error/missing/mistake in the writing of the Typeshed stubs.
I put them at the end of the Python test file in a dedicated method to avoid having them disturb much the readability of this test file.
Do you think they should be removed ? As the list of constructs was specifically provided in the original Jira ticket and in the RSPEC, I was thinking it would be best to make sure and ensure with our test that they were supported. Please let me know otherwise.